### DISCUSSION TITLE
Welcome to FreeCAD Discussions about debugger/code editor!

### DISCUSSION THREAD

**Discussion by mumme74** [mumme74/FreeCAD#14] [General]
## 👋 Welcome!
  This is a example to make a debugger/code editor for FreeCAD, its currently a work in progress.
  * Help wanted to develop further.
  * It's not yet in a very usable state. But perhaps you can help make it better.

  Currently I do this on my freetime, as a hobby. I am not a programmer by profession, but have a passion for coding
  That said I suppose some experienced eyes would spot many thing where to code code could improve.

  Please start discussions here, so we can keep them at the same place.

**Comment by luzpaz** [comment]
Quick question, isn't it preferable to have this discussion on the forum? More visibility would potentially attract contributors

**Reply by mumme74** [reply]
Possibly, I understand your concern. 
Though my coding habits does not fit well to a forum. I tend to be inactive for really long periods of time and have a very focused stint for some weeks occasionally.

If anybody would like to discuss, test ideas and so on in the forum, fine by me! I just don't want to feel obligated to **continuously** engage in that discussion. I am very single thread focus and often other things in my life take that focus.

Focusing hard helps me clear my mind from abstractions and disruptions, of which I have plenty of in my normal life as a teacher. Coding is sort of a therapy/mindfulness for me, I know its kind of weird... I'm not a very social person, sad to say... When I'm in the "zone" I'm hardly reachable. Which means checking forum for updates steal the focus I would like to obtain in the first place. I know it sounds silly but that's the way I'm built... 
If discussions arise on the forum and I'm away teaching future auto technicians (my day job) and focus full time on that for the whole semester. It would probably feel awkward and rude when there is no response on the forum, though admittedly the same situation might arise here...
Sorry if this doesn't make any sense, I would have trouble putting this into words in my native tongue (Swedish).
